### ISSUE TITLE
aria2c error: HTTP response header was bad or unexpected

### ISSUE THREAD

**Issue by nobut** [RobotsAndPencils/xcodes#286]

**Comment by CRoig** [comment]
+1

**Comment by cedricbahirwe** [comment]
Any news on this?

**Comment by mackayn** [comment]
Getting this trying to install 26.1, a bit stuck

**Comment by Lxh-3260** [comment]
> Getting this trying to install 26.1, a bit stuck

Do you solve this problem now? I also encounter this error.
Error code is as follows.
`aria2c error: HTTP response header was bad or unexpected`

**Comment by mackayn** [comment]
> > Getting this trying to install 26.1, a bit stuck
> 
> Do you solve this problem now? I also encounter this error. Error code is as follows. `aria2c error: HTTP response header was bad or unexpected`

No, I didn't. For whatever reason, XCode I got my MAUI iOS app to compile fine  against 26.1.1 (which downloads fine with XCodes) after a Visual Studio patch but the download for 26.1 constantly fails
